ApplicationLibCode/ProjectDataModel/RimRegularLegendConfig.cpp

```diff
@@ -1229,6 +1234,13 @@ void RimRegularLegendConfig::defineUiOrdering( QString uiConfigName, caf::PdmUiO
         uiOrdering.add( &m_colorLegend );
         uiOrdering.skipRemainingFields( true );
     }
+    else if ( uiConfigName == "TableSettings" ) // TODO: UPDATE NAME!
```

Comment:
Find better name, suggested name is full name of Rim-class wanting this config variant.

Reply:
Named `FlagColorsAndMappingModeOnly` to keep same pattern as other "special case" usages